app/models/response/ResponseDeparture.scala

```diff
@@ -42,6 +43,7 @@ object ResponseDeparture {
       routes.DeparturesController.get(departure.departureId).url,
       routes.MessagesController.getMessages(departure.departureId).url,
       departure.movementReferenceNumber,
+      departure.referenceNumber,
```

Comment:
Are we 100% on restoring this to the returned model?

I ask because if we are including this in the response we're going to need a PR for the public API as well